angular/bootstrap/src/index.ts

```diff
@@ -5,16 +5,52 @@ export * from '@agnos-ui/angular-headless';
 
 export * from './agnos-ui-angular.module';
 
-export * from './components/accordion/accordion.component';
-export * from './components/alert/alert.component';
-export * from './components/modal/modal.component';
-export * from './components/modal/modal.service';
-export * from './components/pagination/pagination.component';
-export * from './components/rating/rating.component';
-export * from './components/select/select.component';
-export * from './components/progressbar/progressbar.component';
-export * from './components/slider/slider.component';
-export * from './components/toast/toast.component';
+export type {
+	AccordionApi,
+	AccordionItemContext,
+	AccordionItemProps,
+	AccordionItemState,
+	AccordionItemWidget,
+	AccordionProps,
+	AccordionState,
+	AccordionWidget,
+} from './components/accordion';
+export {createAccordion, getAccordionDefaultConfig} from './components/accordion';
+export * from './components/accordion';
+
+export type {AlertContext, AlertProps, AlertState, AlertWidget} from './components/alert';
+export {createAlert, getAlertDefaultConfig} from './components/alert';
```

Comment:
Agreed... this would not needed anymore once we manage exports like we do in react / svelte.
(so the possibility to do ```import * from '@agnos-ui/headless/config'```)